deploy/cephcsi/image/Dockerfile

```diff
@@ -57,6 +57,8 @@ LABEL maintainers="Ceph-CSI Authors" \
     architecture=${GO_ARCH} \
     description="Ceph-CSI Plugin"
 
+RUN dnf -y install procps-ng
```

Comment:
You really should add a comment here. The `cephcsi` executable does not depend on this, so it looks like bloat for the container-image.